addons/delivery/views/delivery_view.xml

```diff
@@ -22,72 +22,59 @@
             <field name="model">delivery.carrier</field>
             <field name="arch" type="xml">
                 <form string="Carrier">
+                    <header>
+                        <field name="state" widget="statusbar" attrs="{'invisible':['|', ('delivery_type', '=', 'fixed'), ('delivery_type', '=', 'base_on_rule')]}" clickable="True"/>
+                    </header>
```

Comment:
So this field is invisible when type = fixed or type = base_on_rule. But those are the two only states, right ? So when can this field be visible ?